.github/workflows/release.yml

```diff
@@ -14,8 +14,6 @@ jobs:
     steps:
     - name: checkout
       uses: actions/checkout@v4
-      with:
```

Comment:
It caused me an error when I used it (the release wasn't tested- because this merged after the last release)

Reply:
what was the error? 
I think we need the tags for the LD_FLAGS, otherwise it reports an error for `git describe`.
Per https://github.com/actions/checkout/issues/701#issuecomment-1139627817, an alternative would be to add another step right below the checkout that does `git fetch --tags origin`